tests/Oryx.BuildImage.Tests/Php/PhpDynamicInstallationTest.cs

```diff
@@ -44,9 +44,8 @@ public PhpDynamicInstallationTest(ITestOutputHelper output) : base(output)
                     PhpVersions.Composer23Version
                 );
                 data.Add(PhpVersions.Php74Version, imageHelper.GetGitHubActionsBuildImage(), PhpVersions.Composer23Version);
-                data.Add(PhpVersions.Php80Version, imageHelper.GetGitHubActionsBuildImage("buster"), PhpVersions.Composer23Version);
-                data.Add("8.1.4", imageHelper.GetGitHubActionsBuildImage("buster"), PhpVersions.Composer23Version);
-
+                data.Add(PhpVersions.Php80Version, imageHelper.GetGitHubActionsBuildImage("github-actions-buster"), PhpVersions.Composer23Version);
+                data.Add("8.1.4", imageHelper.GetGitHubActionsBuildImage("github-actions-buster"), PhpVersions.Composer23Version);
```

Comment:
Thanks for replacing `buster` by `github-actions-buster` and fixing this! I imagine the test was defaulting to stretch image before.